presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -50,6 +52,7 @@ class AgendaViewModel @Inject constructor(
         fetchAgendaItems(ZonedDateTime.now())
         syncAgendaItems()
         syncFullAgendaItems()
+        synAgendaTasksReminders()
```

Comment:
Since you are calling it just once, and it's a one-liner, you don't really need a method